package-inspector/src/main/java/com/microsoft/inspector/MainActivity.java

```diff
@@ -121,8 +122,14 @@ private void filterAndDisplayPackages(@Nullable final String filterText) {
             @Override
             public void onItemClick(AdapterView<?> adapterView, View view, int position, long l) {
                 try {
+                    String pkgName = packageNames.get(position);
+
+                    if (pkgName.endsWith(ACCOUNT_AFFINITY_SUFFIX)) {
```

Comment:
Why does it have that suffix? seems sort of a strange suffix

Reply:
Any app that has an account type in AccountManager has one put on there so you can differentiate in the list... I'm totally open to changing it to something else

![image](https://user-images.githubusercontent.com/990063/118064640-3ef60c00-b350-11eb-980f-a6bfbc647645.png)

Reply:
![image](https://user-images.githubusercontent.com/990063/118064704-61882500-b350-11eb-91ac-3f91a67d90f9.png)

Reply:
I see. I think this is fine